src/components/BulkLookup.svelte

```diff
@@ -0,0 +1,29 @@
+<script>
+  import { onMount, createEventDispatcher } from "svelte";
+  import { state as authState } from "../auth.js";
+  import { bulkId } from "../couch.js";
+
+  const dispatch = createEventDispatcher();
+  let token = $authState.token;
+  export let metaId = "";
+  let metaIdList = metaId.split("\n");
+  let contentarr = [];
+  let content;
+  let metaList = [];
+  let contentcheck;
+  export let id, db;
+  export let label = "Please provide a label for this Component";
```

Comment:
A lot of these variables don't seem to get used anywhere. I'm also not sure why they're named the way they're named. It seems like `metaId` should just be `id`, and I'm confused as to what the split is doing.